assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -261,14 +262,36 @@ const PaMessageTable: ComponentType<PaMessageTableProps> = ({
               <PaMessageRow
                 key={paMessage.id}
                 paMessage={paMessage}
-                stateFilter={stateFilter}
-                updatePaMessage={updatePaMessage}
+                onEndNow={() => {
+                  setToastProps({
+                    variant: "info",
+                    message: "PA/ESS message has ended, and moved to “Done.“",
```

Comment:
Since we've used an opening quote `“` here, the second quote should be a closing one `”`.

Reply:
Woops, thanks.